tools/image_processing/imagej2/imagej2_analyze_skeleton/imagej2_analyze_skeleton.xml

```diff
@@ -78,18 +78,28 @@
         </test>
     </tests>
     <help>
+
+.. class:: warningmark
+
+The underlying AnalyzeSkeleton Fiji plugin works on binary images, so other image types will
+automatically be converted to binary before they are analyzed.  This step is performed using the
+ImageJ2 **Make Binary** command with the following settings: **Iterations:** 1, **Count:** 1,
+**Black background:** No, **Pad edges when eroding:** No.  If these settings are not appropriate,
+first manually convert the image to binary using the **Convert to binary (black and white) with
```

Comment:
I Greg! I'm in NY at the airport currently, on my way back :(
To you problem, try to rebuild please your image if you are based on :dev.
@afgane have worked a lot on this image during the last 3 days and fixed yesterday this bug: https://github.com/bgruening/docker-galaxy-stable/pull/106

The idea is to test all images automatically with travis :)